### DISCUSSION TITLE
SQLite Support

### DISCUSSION THREAD

**Discussion by xSkeletor** [diced/zipline#395] [Ideas]
It would be great if Zipline could incorporate support for the SQLite database, as it is already utilizing Prisma. Although SQLite may not be the most dependable and efficient production solution, it would be convenient for users like me who don't frequently use Zipline and are the only ones using it. Having an external database seems like extra effort, and I'd prefer everything to be centralized within Zipline, if feasible.

**Comment by diced** [comment]
Unfortunately the way we have it setup, it would be extremely difficult to add this in. Zipline also has a system where it is easy to just up your instance with one command, leaving the frustration out.

**Comment by eden881** [comment]
I really need this too in order to make the jump to Zipline.
It looks great and promising, but I don't want the overhead of yet another Postgres container that lives on my Docker host.
